portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -762,6 +762,72 @@ definition {
 			key_fieldReferenceValue = "SingleSelection1234");
 	}
 
+	@description = "LPS-90435 This is a test verify that the Single selection Form field with non-english site default can not be broken"
+	@priority = "4"
+	test SingleSelectionFormFieldWithNonenglishSiteDefaultCanNotBeBroken {
+		property test.name.skip.portal.instance = "FormsSingleSelectionField#SingleSelectionFormFieldWithNonenglishSiteDefaultCanNotBeBroken";
+
+		task ("Issue 1:The the Single selection Form field with non-english site default can be broken when change the site default language after create a form.") {
+			JSONGroup.addGroup(groupName = "Site Name 1");
+
+			ApplicationsMenu.gotoSite(site = "Site Name 1");
+
+			FormsAdmin.addForm(groupName = "Site Name 1");
+
+			Form.editName(formName = "Form 1");
+
+			Form.editDescription(formDescription = "description 1");
+
+			Form.gotoAddField(
+				fieldPositionNumber = "0",
+				fieldType = "Single Selection");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 1",
+				optionValue = "1");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 2",
+				optionValue = "2");
+
+			Form.save();
+
+			ProductMenu.changeDefaultLanguage(defaultPortalLanguage = "German (Germany)");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Forms");
+
+			FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+			Form.editName(formName = "New Form 1");
+		}
+
+		task ("Issue 2:The the Single selection Form field with non-english site default can be broken when create a form after change the site default language.") {
+			JSONGroup.addGroup(groupName = "Site Name 2");
+
+			ApplicationsMenu.gotoSite(site = "Site Name 2");
+
+			ProductMenu.changeDefaultLanguage(defaultPortalLanguage = "German (Germany)");
+
+			FormsAdmin.addForm(groupName = "Site Name 2");
+
+			Form.gotoAddField(
+				fieldPositionNumber = "0",
+				fieldType = "Single Selection");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 1",
+				optionValue = "1");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 2",
+				optionValue = "2");
+
+			Form.save();
```

Comment:
I thought these two tasks would be similar enough to add them all in on one test, without this much duplication. 
 In order to help this test go more quickly, it's best to split it up, because if the first case is broken then we won't even see this second one.

Let's split these two into two tests, naming the first one: `SingleSelectionFormFieldAddedBeforeDefaultSiteLanguageToggleCanBeLoaded`

And this highlighted second one: `SingleSelectionFormFieldNameAndOptionsCanBeAddedToNonEnglishDefaultSiteLanguageSite`